sympy/physics/vector/frame.py

```diff
@@ -1,5 +1,5 @@
 from sympy.core.backend import (diff, expand, sin, cos, sympify, eye, symbols,
```

Comment:
CI error message is suggesting that this can be amended to
```suggestion
from sympy.core.backend import (diff, expand, sin, cos, sympify, eye,
```

Reply:
Yeah saw it and fixed it with a force push.